Mathlib/Topology/LocalHomeomorph.lean

```diff
@@ -443,6 +447,17 @@ theorem preimage_open_of_open {s : Set β} (hs : IsOpen s) : IsOpen (e.source 
   e.continuousOn.preimage_open_of_open e.open_source hs
 #align local_homeomorph.preimage_open_of_open LocalHomeomorph.preimage_open_of_open
 
+/-- A local homeomorphism is an open map on its source. -/
+lemma isOpenMapOn_source {s : Set α} (hs : IsOpen s) (hse : s ⊆ e.source) : IsOpen (e '' s) := by
```

Comment:
the lemma name should most often match rather closely the statement, instead of trying to convey an informal meaning, because people can invent a lot of informal meaning, while it is much easier to guess the right statement. It could be something like: `isOpen_image_of_subset_source`.